data/data.csv

```diff
@@ -1,3 +1,4 @@
-150,48.9,2
-200,64.7,0.5,1.5
-250,80.5,0.5,1.5
\ No newline at end of file
+0.481,0.002,0.002,200,2,2
```

Comment:
I think at this point it might be worth to actually include a CSV header and just skip the first line during parsing. Otherwise, it's tedious to always look into the documentation when dealing with the data files.